dts/riscv/it8xxx2.dtsi

```diff
@@ -1416,9 +1417,8 @@
 				   &wuc_wu35   /* KSI[5] */
 				   &wuc_wu36   /* KSI[6] */
 				   &wuc_wu37>; /* KSI[7] */
-			gpio-dev = <&gpioc>;
-			pinctrl-0 = <&pinctrl_kso16   /* GPC3 */
-				     &pinctrl_kso17>; /* GPC5 */
+			kso16-gpios = <&gpioc 3 (GPIO_OPEN_DRAIN | GPIO_PULL_UP)>;
+			kso17-gpios = <&gpioc 5 (GPIO_OPEN_DRAIN | GPIO_PULL_UP)>;
```

Comment:
I've checked your reference manual, https://www.ite.com.tw/uploads/product_download/IT81202_B_V0.3.2_20201030.pdf, and I see all peripherals have in reality a single pinmux selection, so after all, feel free to move `pinctrl-` entries to the SoC files if you want. All boards will be in practice force to use the same selection anyway. It's the first SoC that I see like this :-)

Reply:
I think that pinctrl-n are written in the board DT file for consistency. Thanks for your suggestions and patient review.